demo/run-osm-demo.sh

```diff
@@ -125,24 +127,30 @@ else
       --enable-prometheus-scraping="$ENABLE_PROMETHEUS_SCRAPING" \
       --envoy-log-level "$ENVOY_LOG_LEVEL" \
       --timeout=90s \
+      --enable-routes-v2-experimental="$ROUTES_V2"
       $optionalInstallArgs
 fi
 
-./demo/configure-app-namespaces.sh
-
-./demo/deploy-apps.sh
+if [ "$ROUTES_V2" = "true" ]; then
```

Comment:
This script is already getting way too complicated. Would you mind creating a separate demo script to work with routes-v2 under `experimental/`? It will be easy to just switch over and deprecate this one later on.

Reply:
I have reverted the change in this file and added a new script to the experimental folder.